tests/mpd/test_commands.py

```diff
@@ -111,12 +115,16 @@ def test_function_has_no_arguments_fails(self):
 
     def test_function_has_required_and_varargs_fails(self):
         with self.assertRaises(TypeError):
-            func = lambda context, required, *args: True
+            def func(context, required, *args):
+                True
```

Comment:
I guess this and the next function could simply have `pass` as the body. With `True` I was wondering a bit if the function body had any relevance to the test.